src/main/java/net/imagej/ops/sieve/highSieve/DefaultHighSieve.java

```diff
@@ -0,0 +1,25 @@
+package net.imagej.ops.sieve.highSieve;
+
+import net.imagej.ops.Ops;
```

Comment:
The order of the imports is non-standard. If you are using Eclipse @rimadoma, you could _Organize Imports_ with the latest [ImageJ Eclipse preferences](https://github.com/imagej/imagej/blob/master/config/eclipse-preferences.epf) active to do that.